<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogContourLevels</class>
 <widget class="QDialog" name="DialogContourLevels">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>389</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>265</width>
    <height>389</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>265</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter Contour Levels</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="3,3,1,1,1,1,1,1,1">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Level Creation Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radioButton_exactLevels">
        <property name="text">
         <string>Exact levels</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_approximateLevels">
        <property name="text">
         <string>Approximate levels for nice values</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_rangeDistribution" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_rangeDistribution">
        <property name="title">
         <string>Range Distribution</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_range0">
           <property name="text">
            <string>Min, max, and number of levels</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_range1">
           <property name="text">
            <string>Min, max, and delta</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_range2">
           <property name="text">
            <string>Exponential distribution</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_minimunLevel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_minimunLevel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Minimum level:    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_minimunLevel"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_maximumLevel" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_maximumLevel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Maximum level:    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_maximumLevel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_numberofLevels" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_numberOfLevels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Number of levels: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_numberOfLevels">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_delta" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_delta">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Delta:            </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_delta">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_resetRange">
       <property name="text">
        <string>Reset Range</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_space">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>5</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancle">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancle</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_resetRange</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>resetRange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_exactLevels</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickExactLevels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_approximateLevels</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickApproximateLevels()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>38</x>
     <y>3</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_range0</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickRangeDistribution0()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_range1</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickRangeDistribution1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_range2</sender>
   <signal>clicked()</signal>
   <receiver>DialogContourLevels</receiver>
   <slot>onClickRangeDistribution2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>161</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resetRange()</slot>
  <slot>onClickOK()</slot>
  <slot>onClickExactLevels()</slot>
  <slot>onClickApproximateLevels()</slot>
  <slot>onClickRangeDistribution0()</slot>
  <slot>onClickRangeDistribution1()</slot>
  <slot>onClickRangeDistribution2()</slot>
 </slots>
</ui>
